### ISSUE TITLE
8/4K videos are showing as 1080 and sometimes 720p

### ISSUE THREAD

**Issue by realrobd** [yuliskov/SmartYouTubeTV#820]
Watching a 4k video it's obvious it's not 4k.
When checking under "quality" setting, there is no option for 4k.
Switching over to the regular YouTube app, it plays properly.

**Comment by jarrard** [comment]
yeah I get this on my TV, previous one I could watch higher res fine but now its 720p max.

**Comment by realrobd** [comment]
> yeah I get this on my TV, previous one I could watch higher res fine but now its 720p max.

This is the same tv. 
The problem started with the last app update.

**Comment by szeller1111** [comment]
Same issue occured on my Sony Z9F since last update

**Comment by jarrard** [comment]
SmartYTNext seems pretty decent, works allot better. Only issue is it doesn't sequentially follow a ordered playlist with numbered  videos.

**Comment by Dozer58** [comment]
Same for me, only get 720p now, Is there a download like for old version?
I don't like smartYTnext as is doesn't save search history so can't quickly search for the same things with one click and it's also not showing videos related to what I watch.

**Comment by jarrard** [comment]
> Same for me, only get 720p now, Is there a download like for old version?
> I don't like smartYTnext as is doesn't save search history so can't quickly search for the same things with one click and it's also not showing videos related to what I watch.

There are older versions in the releases section. I think Smart.YouTube.TV.6.17.730.apk is previous and showed the older pro launcher options. 

The only issue I have with Next is that it suggests the wrong videos with autoplay, it often goes backwards in a playlist instead of forward. For example playing VIDEO-10 will then autoplay VIDEO-9 or some other thing instead of VIDEO-11 like the previous version did. 

Might be best to make a bug report about NEXT separate. But for me it performs allot better on my crap T95z android player.
